src/components/SelectionList/ExpenseItemHeader.tsx

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import {View} from 'react-native';
+import Button from '@components/Button';
+import Icon from '@components/Icon';
+import * as Expensicons from '@components/Icon/Expensicons';
+import useStyleUtils from '@hooks/useStyleUtils';
+import useTheme from '@hooks/useTheme';
+import useThemeStyles from '@hooks/useThemeStyles';
+import variables from '@styles/variables';
+import type {SearchAccountDetails} from '@src/types/onyx/SearchResults';
+import UserInfoCell from './UserInfoCell';
+
+type ExpenseItemHeaderProps = {
+    participantFrom: SearchAccountDetails;
+    participantTo: SearchAccountDetails;
+    buttonText: string;
+    onButtonPress: () => void;
+};
+
+function ExpenseItemHeader({participantFrom, participantTo, buttonText, onButtonPress}: ExpenseItemHeaderProps) {
```

Comment:
This is a good component name, however we use it only for narrow (`!isLargeScreenWidth()`) screens.
So maybe change the name to something like `ExpenseItemHeaderNarrow` or similar? Do you think it would be helpful or maybe not?